### DISCUSSION TITLE
第6节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#429] [笔记]
第6节课笔记(3班)

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/18/mmtutorial-6-note/

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/678904358

**Comment by xiexiaoshinick** [comment]
[[笔记和作业]6.OpenCompass 大模型评测 - 知乎 (zhihu.com)](https://zhuanlan.zhihu.com/p/679047918)

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/679063645

**Comment by 443752009** [comment]
http://t.csdnimg.cn/k1N5i

**Comment by Sampson425** [comment]
https://blog.csdn.net/m0_49908454/article/details/135748739

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/47vgx

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/mr7xlfgpm8c2dpd8?singleDoc# 《OpenCompass评测笔记》

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/679328145

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135856799
